creds/src/dlog.rs

```diff
@@ -128,8 +128,8 @@ impl<G: Group> DLogPoK<G> {
         // serialize and hash the bases, k and y
         let dl_verify_timer = start_timer!(|| format!("DlogPoK verify y.len = {}", y.len()));
         let mut ts: Transcript = Transcript::new(&[0u8]);
-        let pm = pm.unwrap_or(b"");
-        add_to_transcript(&mut ts, b"presentation_message", &pm);
+        let pm = context.unwrap_or(b"");
```

Comment:
Should we change the pm variable name to contextBytes?

Reply:
done